README.md

```diff
@@ -108,15 +108,29 @@ $usernameValidator->validate('alexandre gaigalas'); //false
 $usernameValidator->validate('#$%');                //false
 ```
 
+### Exception Types
+
+* `Respect\Validation\Exceptions\NestedValidationExceptionInterface`:
+    * Use when calling `assert()`.
+    * Interface has three methods: `getFullMessage()`, `findMessages()`, and `getMainMessage()`.
+* `Respect\Validation\Exceptions\ValidationExceptionInterface`: 
+    * Use when calling `::check()`.
+    * All `Respect\Validation` validation exceptions implement this interface.  
+    * Interface has one method: `getMainMessage()`;
+* `Repect\Validation\Exceptions\ExceptionInterface`: 
+    * All `Respect\Validation\Exceptions` implement this interface.  
+ 
 ### Informative Exceptions
 
 When something goes wrong, Validation can tell you exactly what's going on. For this,
 we use the `assert()` method instead of `validate()`:
 
 ```php
+import Respect\Validation\Exceptions\NestedValidationExceptionInterface
```

Comment:
hehehe, I was wondering if it was Python, C, Java or what :stuck_out_tongue_closed_eyes: